src/telemetry/metrics/http/tower_layer.rs

```diff
@@ -181,6 +182,10 @@ impl HttpCallMetricTracker {
 			res.push(KeyValue::new("server.address", host.to_owned()));
 		}
 
+		if TELEMETRY_NAMESPACE.trim().is_empty() {
```

Comment:
This should negate right?
```suggestion
		if !TELEMETRY_NAMESPACE.trim().is_empty() {
```

Reply:
Yes indeed. I was just testing it and noticed it as well.